src/extras/webgl/TilingRenderer.js

```diff
@@ -0,0 +1,145 @@
+import * as core from '../../core';
+import { WRAP_MODES } from '../../core/const';
+
+const glslify = require('glslify'); // eslint-disable-line no-undef
+
+const tempMat = new core.Matrix();
+const tempArray = new Float32Array(4);
+
+/**
+ * WebGL renderer plugin for tiling sprites
+ */
+export class TilingRenderer extends core.ObjectRenderer {
```

Comment:
Can we rename this to `TilingSpriteRenderer` to be consistent with the other names of our renderers?